src/services/ui/src/pages/Home.tsx

```diff
@@ -1,3 +1,142 @@
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+              ],
+              [
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+              ],
+              [
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+              ],
+              [
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+              ],
+            ],
+          },
+          {
+            buttonText: "Section",
+            columns: [
+              [
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+              ],
+            ],
+          },
+        ]}
+        secondaryComponent={
+          <ActionsMenu
+            links={[
+              {
+                href: "",
+                iconName: "person",
+                onClick: function noRefCheck() {},
+                target: "_blank",
+                text: "Manage Profile",
+              },
+              {
+                href: "",
+                iconName: "people",
+                onClick: function noRefCheck() {},
+                target: "_blank",
+                text: "Request Role Change",
+              },
+              {
+                href: "",
+                iconName: "logout",
+                onClick: function noRefCheck() {},
+                target: "_blank",
+                text: "Log Out",
+              },
+            ]}
+            name="My Account"
+          />
+        }
+      />
+      <Footer emailAddress="storybook-test@cms.hhs.gov" />
```

Comment:
there is prob a way to have the footer always on the bottom of the page. Also, we should prob shange the emaill address here to something more appropriate.

Reply:
i fixed it